src/loading-spinner/Loading.tsx

```diff
@@ -0,0 +1,28 @@
+import { GlobalStyles } from "../styles";
```

Comment:
Can you move this into `design-system`? Try to move all globally-used visual things there.

Reply:
Good call, done!